graphql/directive_test.go

```diff
@@ -23,8 +24,7 @@ func setupPermissions(t *testing.T) {
 	require.NoError(t, env.DB().Drop(ctx))
 
 	// Create scope and role collection to avoid RoleManager from trying to create them in a collection https://jira.mongodb.org/browse/EVG-15499
```

Comment:
This (preexisting) comment threw me for a loop. I think he meant "from trying to create them in a transaction"?

Reply:
Yeah, the ticket says it's because it's trying to insert docs into nonexistent collections in a transaction. I fixed the comments.

Reply:
ty!